src/WebJobs.Script.WebHost/Models/HostAssignmentContext.cs

```diff
@@ -36,6 +35,13 @@ public class HostAssignmentContext
         [JsonProperty("Secrets")]
         public FunctionAppSecrets Secrets { get; set; }
 
+        // This will be true for dummy specialization calls to pre-jit specialization code.
+        // For warmup requests the Run-From-Pkg appsetting will point to a local endpoint that
+        // returns 200 to ensure downloading app contents succeeds.
+        // All the other fields will be empty.
+        [JsonProperty("isWarmupRequest")]
+        public bool IsWarmupRequest { get; set; }
```

Comment:
Can we get into a case in the rollout of this where this value isn't being sent yet to an updated host, meaning it'll try to do a full assign?

Reply:
Good question, there are 2 clients that makes this call, the mesh service and antares.
We want all requests from Antares to always do the full assign. Antares will never set EncryptedHostAssignmentContext.WarmUp or HostAssignmentContext.IsWarmupRequest. So none of the requests from antares will be treated as a warmup request with or without this PR.

For the other client, yes as soon as this PR is merged the corresponding PR in docker should also be merged. I called this out in the linked PR so we dont forget this
https://github.com/Azure/azure-functions-docker-private/pull/89#issuecomment-668799586

In case we miss this it will completely brick the image since no site assignment can take place and so it will fail in an obvious that will fail our OGFs and sanity tests.